### DISCUSSION TITLE
206

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#192] [General]
# 潮流周刊第206期 - 标致的狗

封面图拍摄于外面看到的一只小狗，被圈养在围栏里面，非常标致、毛发很好，甚至让它坐着就很乖坐着，挺可爱。

https://weekly.tw93.fun/posts/206-%E6%A0%87%E8%87%B4%E7%9A%84%E7%8B%97

**Comment by zyzhang0417** [comment]
豆瓣的链接放错了，不过放了我的最爱，表示原谅。

**Reply by tw93** [reply]
哈哈哈 好

**Comment by lovingchina** [comment]
我刚买了 文石 Leaf3C 彩色墨水屏，对于纯文字阅读和黑白屏没啥区别，但是看图书、漫画、公众号长文章的时候，彩色屏就明显很舒服了

**Comment by MikuSugar** [comment]
我也是kindle换成了掌阅的 ireader color7,传书体验比Kindle好太多了，外观和你这个一样，彩色的看漫画也安逸

**Reply by tw93** [reply]
主要是我感觉直接微信读书可以解决大部分问题

**Comment by mrdream24** [comment]
我用的是掌阅 Ocean2，确实很不错。只是不耐摔，要小心！

**Reply by tw93** [reply]
哈哈 好的

**Comment by professorLea** [comment]
只花了 15 块买了可以使用 10 小时
--

**Reply by professorLea** [reply]
你这是买了哪个套餐哈

**Reply by tw93** [reply]
最便宜的那个，当时还有一个优惠券...

**Comment by advancejar** [comment]
天翼云电脑也很不错，还可以用网页来访问，功能稍受限。
我用kindle看英文，目前国内没有阅读器能导出查的生词和所在例句。

**Reply by tw93** [reply]
下次试试去，我还没有开始看英文的书

**Comment by gaosunhao** [comment]
我的阅读器基本都是咸鱼淘的，在用的是文石nova3 7.8寸。之前先后用过墨案、小米多看、kindle，下次试试掌阅

**Reply by tw93** [reply]
哈哈哈 赞折腾
